package.json

```diff
@@ -6,7 +6,7 @@
   "repository": "https://github.com/metabase/metabase",
   "license": "private",
   "engines": {
-    "node": ">=16",
+    "node": "^16",
```

Comment:
I assumed this is only for informative purposes.
Please let me know if this change should be done in a separate PR.

Reply:
should be good

Reply:
It looks like it's breaking [some tests](https://github.com/metabase/metabase/actions/runs/5010843765/jobs/8981087216?pr=30823) though:

> Run yarn build-static-viz
> yarn run v[1](https://github.com/metabase/metabase/actions/runs/5010843765/jobs/8981087216?pr=30823#step:4:1).22.19
> error metabase@0.0.0: The engine "node" is incompatible with this module. Expected version "^16". Got "1[8](https://github.com/metabase/metabase/actions/runs/5010843765/jobs/8981087216?pr=30823#step:4:9).16.0"
> error Commands cannot run with an incompatible environment.
> info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
> Error: Process completed with exit code 1.

I'll revert this line and make sure this PR is only about markdown updates.